config/mgc-install-guide/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
+  - ../default
+  - namespace.yaml
+  - gatewayclass.yaml
```

Comment:
We should able to remove this. Installation of mgc should be possible in all scenarios by applying the `config/default` kustomization, now that you have updated the namespace in that to a sensible default, it will always be correct and you don't need the namespace.yaml here.

Reply:
This creates the `multi-cluster-gateways` namespace, which is used later in the guide as the namespace for the `ManagedZone`. Do you think I should delete this directory and create this namespace separately or use a different namespace for the `ManagedZone`?

Reply:
Yeah, so IMO i don't think we should create that ns as part of any kustomization. This is a namespace that the user is going to create stuff in and should really be whatever they want to call it. In fact, i don't think anything that the user is expected to create should be created via kustomize and should be called out in the docs explicitly  as resources they should apply (You are already doing this in your doc for MZs and Issuers).